impactlab_tools/os/config.py

```diff
@@ -0,0 +1,24 @@
+import sys
+import yaml
+
+def getConfigDictFromFile(filePath):
+    with open(filePath, 'r') as f:
+        confDict = yaml.load(f)
+    return confDict
+
+def getConfigDictFromSysArgv():
+    with open(sys.argv[1], 'r') as f:
+        confDict = yaml.load(f)
+
+        for key in serverConfig:
+            if key not in confDict:
+                confDict[key] = serverConfig[key]
+
+        return confDict
+
+## Server config
+
+serverSpecificYML = "../server.yml"
+
+serverConfig = getConfigDictFromFile(serverSpecificYML)
```

Comment:
Please make this a function, accepting serverSpecificYML as an argument (default value of "../server.yml" is ok).

Let's avoid executing code on module load if this is going to be bundled in a larger package.